### PR TITLE
Image sizes

### PR DESCRIPTION
**Pull request by Tanner-Scadden**
- Added in the ability to pass down image/devices sizes to the component. 
- Added in taking the loader from context if it exists.

### CODE REVIEW

File: examples/movies/src/components/image/image-config.ts [hunk 1]
```diff
@@ -1,94 +1,92 @@
...
 
   /** @see [Acceptable formats](https://nextjs.org/docs/api-reference/next/image#acceptable-formats) */
-  formats: ImageFormat[]
+  formats: ImageFormat[];
 
   /** @see [Dangerously Allow SVG](https://nextjs.org/docs/api-reference/next/image#dangerously-allow-svg) */
-  dangerouslyAllowSVG: boolean
+  dangerouslyAllowSVG: boolean;
 
   /** @see [Dangerously Allow SVG](https://nextjs.org/docs/api-reference/next/image#dangerously-allow-svg) */
```
[the remark this answers is not in the document]
Comment (Tanner-Scadden) [reply examples/movies/src/components/image/image-config.ts:67]: @davedbase

**Comment by Tanner-Scadden** [comment]
> 

Would be really nice to get added. David is leading this component addition for solid-start. Maybe we can put together a feature list that would be nice to have once we have the port completed from nextjs.

**Comment by davedbase** [comment]
> Maybe not in the scope of this PR, but a plugin for blurhash integration might be useful, too.

I was working on this but there were some ESM importing issues with the blurhash package when I tried. @atk if you want to add that, it would absolutely fantastic. Maybe we should look at vite-plugin-blurhash in case it can help us along as well.

File: examples/movies/src/components/image/image-config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (davedbase) [reply examples/movies/src/components/image/image-config.ts:67]: > Comment for content security policy is wrong.

Thanks, I'll have a closer look at this. The component needs quite a bit of work.

**Pull request merged**